Library/ViewModels/PledgeViewModel.swift

```diff
@@ -1152,10 +1152,8 @@ private func checkoutPropertiesData(
 
   let staticUsdRate = Double(createBackingData.project.stats.staticUsdRate)
 
-  let pledgeTotalUsdCents = pledgeTotal
-    .multiplyingCurrency(staticUsdRate)
-    .multiplyingCurrency(100.0)
-    .rounded()
+  // Two decimal places to represent cent values
+  let pledgeTotalUsd = rounded(pledgeTotal.multiplyingCurrency(staticUsdRate), places: 2)
```

Comment:
This variable is used in a core flow for checkout. Can we add tests for the way we are using it if we don't have them already

Reply:
The fix here entails changing the original representation of cents into a dollar + cents value (e.g. 1015 cents -> 10.15). It turns out that we already have a handful of tests written in the `PledgeViewModelTests` file. A lot of them broke after the fix which made sense because our templates had cent values that all needed to be updated. I want to note, however, that the change here is specifically occurring in the `checkoutPropertiesData` function which is only used for events/tracking and not rendering components of the UI, from my observation. 

After I fixed the templates and tests, it appears our coverage for both this method and the entire `PledgeViewModel` are good.

Reply:
Amazing! Thank you!